composer.json

```diff
@@ -17,7 +17,7 @@
         }
     ],
     "require": {
-        "php": "~8.1.0",
+        "php": "~8.1.0 || ~8.2.0",
```

Comment:
8.2 should also be in CI before merge

Reply:
https://github.com/lcobucci/jwt/blob/2fecef3210030c1bfaead33116228118b7ca9cc0/.github/workflows/phpunit.yml#L77